jobs/kubernetes-system-specs/templates/config/kubedns-rc.yml

```diff
@@ -30,11 +39,17 @@ spec:
         k8s-app: kube-dns
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
     spec:
+      tolerations:
+      - key: "CriticalAddonsOnly"
+        operator: "Exists"
+      volumes:
+      - name: kube-dns-config
+        hostPath:
```

Comment:
What was this issue you saw using configMap instead of hostPath , mentioned in this [commit](https://github.com/pivotal-cf-experimental/kubo-release/pull/32/commits/dc792e7228902685a6891194e42c1573b2799fac)?